OpenGov-OpenData/ckanext-harvest
# [OD-1072] Add a link to the harvest page in the account masthead

## [Ticket](https://opengovinc.atlassian.net/browse/OD-1072)

## Description
The current version of the master branch is between [1.1.0](https://github.com/OpenGov-OpenData/ckanext-harvest/tree/v1.1.0) and [1.1.1](https://github.com/OpenGov-OpenData/ckanext-harvest/tree/v1.1.1). This PR updates the harvester to version 1.1.1 and adds a link to the harvest page in the account masthead.

Version v1.1.1 of the harvester added a feature to email sysadmins if a harvest job fails and some improvements to the format of the code.

## Testing
- Install this PR
- Restart the supervisor service
- Check that the harvest link is in the header
- Check that harvest extension can still works as expected. The dcat extension should be able to harvest http://bostonopendata.boston.opendata.arcgis.com/data.json
